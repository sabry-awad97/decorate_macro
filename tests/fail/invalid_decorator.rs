use decorate_macro::decorate;

#[decorate(123)] // Invalid decorator path
fn test_function(x: i32) -> i32 {
    x
}

fn main() {}
