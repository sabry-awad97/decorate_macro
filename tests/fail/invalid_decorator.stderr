error: expected identifier
 --> tests/fail/invalid_decorator.rs:3:12
  |
3 | #[decorate(123)] // Invalid decorator path
  |            ^^^
